### DISCUSSION TITLE
Enable or disable config for any field

### DISCUSSION THREAD

**Discussion by RajGM** [AntlerVC/firetable#989] [Support / Q&A] [answered]
Hi @notsidney 
I was looking to enable the column config to enable the image compression feature for the image field, but I cannot locate the modal/config to do so.
Is there any generic pathway to enable or disable config for any column?

**Comment by BeeBombshell** [comment, marked as answer]
@RajGM, Rowy supports compression by default for the image field. There is an open issue to add the functionality for the users to turn it off in the Image settings.  Here's a link to the issue: https://github.com/rowyio/rowy/issues/539. 
If you're interested in working on it, feel free to comment and somebody will assign it to you.

**Reply by RajGM** [reply]
Thanks for the info, @BeeBombshell 
I will have a look at it.
